spec/lib/dispatch-rider/notification_services/file_system/channel_spec.rb

```diff
@@ -11,7 +10,7 @@
   subject { described_class.new(path) }
 
   describe "#publish" do
-    let(:message){ {:subject => "foo", :body => "bar"}.to_json }
+    let(:message) { {:subject => "foo", :body => "bar"}.to_json }
```

Comment:
Use the new Ruby 1.9 hash syntax.<br>Space inside { missing.<br>Space inside } missing.